hashtagsv2/hashtags/forms.py

```diff
@@ -10,6 +12,9 @@ class SearchForm(forms.Form):
 		widget=forms.TextInput(attrs={'placeholder': 'Enter a hashtag',
 			'style': 'width:100%'}))
 
+	search_type = forms.CharField(label="Type:",
```

Comment:
This field should have `required=False`, to enable the tool to continue working even when `search_type` isn't in the search request (such as when clicking on a Top Tag).